notebooks/notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %run "../src/pipelines/graphs_section.py"
</pre><p>If this script is too long to be explicit in the notebook, I suggest explaining the general methods it uses to generate the plots. Also the plot descriptions are a bit difficult to read, increasing the font size might help, alternatively adding some space between figures could improve readability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/repro-challenge-team-3/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Space had been added.</p>

Reply:
closing conversation, resolved